### ISSUE TITLE
the testing code does not work

### ISSUE THREAD

**Issue by puppet101** [jiaxiangshang/MGCNet#4]
Hi, I used the testing code in the test_unsupervise.py, but some errors occurred. Is the testing code unfinished? 
Thank you！

**Comment by jiaxiangshang** [comment]
Yes, the test code is unfinished, I try to finish it before August 15.

**Comment by puppet101** [comment]
Hi, @jiaxiangshang 
Thank you for finishing the testing code. But the result is a little different from which it shows in the demo.
For example, the color of this image is a little strange, and the shape is also not as perfect as it is in your demo.

![image04275_mulPoses](https://user-images.githubusercontent.com/9941762/90753189-dde26000-e30a-11ea-825b-858785dbaecf.jpg)

**Comment by jiaxiangshang** [comment]
Really thanks for your quick feed back.
The color is sure of a problem. I will first fix this and tell you.
For the shape, I will test more images and see if this is because of the new model that I trained with this release version code.

**Comment by puppet101** [comment]
Thanks a lot for your excellent work!

**Comment by jiaxiangshang** [comment]
The color bug now fixed.
For the shape rendered on images, the overlay result alignment mainly depends on the camera pose rather than the face shape, the face shape can be visualized in meshlab that is still good.
For the camera pose, I will check the model I uploaded, I will update tomorrow.

**Comment by puppet101** [comment]
Great! Thank you again for your help!
